### ISSUE TITLE
Length assessment bug [ROI]

### ISSUE THREAD

**Issue by silmarrr** [horosproject/horos#505]
Hi everyone,
while measuring CAG images the bug comes out:
![Bug](https://user-images.githubusercontent.com/56556719/66769279-feed4f80-eeb4-11e9-971a-6212d900c32b.jpg)
is there any way to fix this or any way around?
Thanks

**Comment by TAVR-man** [comment]
Problem present with all ROI measurements since switching to Catalina and still present in v4.0.0 RC2
<img width="532" alt="image" src="https://user-images.githubusercontent.com/56940762/67438458-745ecb80-f5eb-11e9-8a71-d289ffcb1fab.png">

**Comment by nico87mi** [comment]
I have the same problem with a MacBook Pro 2018 after switching to Catalina

**Comment by mamp2752** [comment]
tengo el mismo problema com MacBook Pro 2017 luego de instalar catalina

**Comment by aglv** [comment]
I cannot reproduce this on my 2018 MPB.

**Comment by FabriceHx** [comment]
> I cannot reproduce this on my 2018 MPB.

Does it work with OsiriX ?

**Comment by TAVR-man** [comment]
Works normally in OsiriX

**Comment by silmarrr** [comment]
Yeah, works good.

**Comment by silmarrr** [comment]
TAVR-man, doing some segmentation in TAVR field?

**Comment by TAVR-man** [comment]
Using ProsizeAV plug in for all our measurements. Very quick and get accurate implantation angles. Prefer it to 3mensio

**Comment by silmarrr** [comment]
For TAVR, check out FEops HEARTguide from Belgium, seems interesting.

**Comment by JeanClaude2** [comment]
@FabriceHx Yes, it works in OsiriX.

**Comment by aglv** [comment]
> > I cannot reproduce this on my 2018 MPB.
> 
> Does it work with OsiriX ?

Turns out, I can reproduce it. I'll look into it. But I dunno about OsiriX.

**Comment by Hiroaki-Inomata** [comment]
Hi,

We have reproduced it and maybe solved this issue.
See: https://github.com/air-h-128k-il/HorliX/issues/17#issuecomment-551342830

**Comment by Hiroaki-Inomata** [comment]
By the way, OsiriX **MD** works well on Catalina?

**Comment by FabriceHx** [comment]
> By the way, OsiriX **MD** works well on Catalina?

Yes, without any problems, including 3D rendering & scrolling through images (much slower in Horos).

**Comment by fvpolpeta** [comment]
Hi there -

Release of Horos 4.0.0 is imminent and it includes a solution for this issue.

Please, check the link below and inform your results:

https://horosproject.org/horos-content/Horos4.0.0RC3.dmg

Thanks

Fauze
